gpkit/tests/t_examples.py

```diff
@@ -76,6 +76,9 @@ def test_simpleflight(self, example):
             sol_rat = sol["sensitivities"]["constants"][key]/consenscheck[key]
             self.assertTrue(abs(1-sol_rat) < 1e-2)
 
+    def test_unbounded(self, example):
+        pass
```

Comment:
this gets run for all solvers currently -- we need better control over this -- see #258